spec/features/edit_page_spec.rb

```diff
@@ -9,6 +9,14 @@
     expect(page).to have_header("Edit #{customer.name}")
   end
 
+  it "has button to view customer details" do
+    customer = create(:customer)
+
+    visit edit_admin_customer_path(customer)
+
+    expect(page).to have_link("Show #{displayed(customer)}")
```

Comment:
This may be for my own reference, but - where is this mysterious `displayed` method defined? Don't see it in this spec, or rails_helper. ¯\\\_(ツ)\_/¯